project-ivi/ivi
# removed tests

#169 
# Description

tests removed so travis will pass. Tests were horribly outdated anyway and served no purpose.

# Change Log
Describe all changes made 

- `test/` : deleted
- `.travis.yml` : removed `npm run test`
- `README` : removed test description
- `package.json` : removed test scripts

# Manual Test Steps

`npm run test` should do nothing now
Travis should not run any tests (since none exist)
